yarn.lock

```diff
@@ -5172,7 +5172,7 @@ eslint-plugin-sonarjs@^0.5.0:
   resolved "https://registry.yarnpkg.com/eslint-plugin-sonarjs/-/eslint-plugin-sonarjs-0.5.0.tgz#ce17b2daba65a874c2862213a9e38e8986ad7d7d"
   integrity sha512-XW5MnzlRjhXpIdbULC/qAdJYHWw3rRLws/DyawdlPU/IdVr9AmRK1r2LaCvabwKOAW2XYYSo3kDX58E4MrB7PQ==
 
-eslint-plugin-va-enzyme@./script/custom-eslint/va-enzyme:
+eslint-plugin-va@./script/eslint-plugin-va:
   version "1.0.0"
```

Comment:
Should we be versioning this package?

Reply:
yes, I will but I want to first release the version `1.0.0` before start adding new changes or new rules